src/components/element-editor/basic-details/dropdown-items/dropdown-items.tsx

```diff
@@ -2,8 +2,19 @@ import { Form } from "@lifesg/react-design-system/form";
 import { Text } from "@lifesg/react-design-system/text";
 import { PencilIcon } from "@lifesg/react-icons/pencil";
 import { PlusIcon } from "@lifesg/react-icons/plus";
+import { isEmpty } from "lodash";
```

Comment:
use sub-directory import to reduce file size
```suggestion
import isEmpty from "lodash/isEmpty";
```